lib/ReactViews/ExplorerWindow/Tabs/MyDataTab/AddData.jsx

```diff
@@ -150,6 +150,7 @@ const AddData = createReactClass({
               selectOption={this.selectLocalOption}
               matchWidth={true}
               theme={dropdownTheme}
+              height="150px"
```

Comment:
Should this be part of the theme, rather than a separate prop? (sorry, nitpicking. it probably doesn't matter)

Reply:
I hadn't thought about that to be honest, I'm not sure what the rationale is between stuff that goes into the theme prop vs what gets passed down as its own prop.